### DISCUSSION TITLE
Batterie-Spannung abhängiges Dynamisches Power Limit möglich?

### DISCUSSION THREAD

**Discussion by swingstate** [hoylabs/OpenDTU-OnBattery#351] [General]
Ich habe nun 2x25,6V 100Ah parallel an einem HM-1500. Alles Bestens soweit, allerdings ist mir aufgefallen, dass die Batteriespannung bei größerer dauerhafter Last schnell auf unter 25,6V (bei 900W Dauerlast nach ca. 40 - 50Ah, genaue Zahlen muss ich nochmal messen), so dass der Inverter dann abschaltet - Das ist korrekt, ich habe das Stop Limit auf 25,6 gesetzt. Nach dem Abschalten ging die Spannung wieder auf 26,1V und ich habe den WR mit niedrigerem DPL manuell neugestartet. 

Bekannt ist ja, dass man bei konstant niedriger Last mehr aus der Batterie ziehen kann (in meinem Fall ca. 160Ah bis ich bei 20% SOC bin) bis die Spannung einbricht.

Meine Frage ist nun, kann man das DPL dynamisch anpassen (lassen), so dass die Batterie möglichst lange bei einer Spannung über einem definierten Wert bleibt? Man die Frage überhaupt Sinn? :)

**Comment by schlimmchen** [comment]
> Ma[cht] die Frage überhaupt Sinn? :)

Ja, und das ist bereits implementiert. Schau dir den "DC Voltage - Load correction factor" bzw. "DC Spannung - Lastkorrekturfaktor" in den DPL Einstellungen an. Mein Wert liegt bei 0.0005 und ist empirisch bestimmt (und inzwischen obsolet, weil ich den State of Charge inzwischen aus dem BMS lesen kann).

**Comment by spcqike** [comment]
wie @schlimmchen schreibt, gibt es das bereits.

der Faktor ist für die Leitungsverluste (Kabel, Schraubverbindungen, Busbars, ....) super. 
Aber er sollte meiner Meinung nach nicht dafür benutzt werden den Spannungsabfall, der durch den internen Widerstand der Batterie hervorgerufen wird, "weg zu korrigieren" um die Batterie tiefer zu entladen. also vorsichtig und genau sein beim Eintragen.

Selbstverständlich bricht die Batterie bei höheren Strömen tiefer/schneller ein und erreicht damit andere (kleinere) nutzbare Entladekapazitäten. Gute Diagramme zu einzelnen Zellen findet man hier

https://lygte-info.dk/review/batteries2012/Enerpower%2026650%203.2V%203000mAh%20(Blue)%20UK.html

Da sieht man aber auch, dass eine einzelne 3Ah Zelle zwischen ~2,6Ah (30A Entladestrom -> ~100W) und ~2,8Ah (0.2A Entladestrom -> 1W) hat. also lediglich eine Abweichung von ~7%.

Die prismatischen zellen, die in 100Ah LiFePo4 Batterien verbaut werden, können jedoch mit weitaus größeren Strömen entladen werden, als die oben verlinkte kleine Rundzelle, ohne dabei signifikant einzubrechen.

Wenn du kannst:
Entlade die Batterie mit so viel Strom wie du kannst, also deine 1.500W, und miss den Spannungsabfall vom Batterieanschluss bis zum Wechselrichter. (wahrscheinlich brauchst du dafür ein MC4-Y-Verbinder direkt am Inverteranschluss, sodass du die dort anliegende Spannung mit einem Multimeter messen kannst)

Bei 1.500W und 25,6V Batteriespannung müssten so ~ 59A fließen. Das kann der HM-1500 ja gar nicht. 
Wenn du ihn also auf 1.500W (offen) einstellst, und er zieht was er kann, sollten das 46A (4x11,5A oder?) ziehen und damit ~1200W einspeisen. 
Wenn du dabei dann einen Spannungsabfall von 1V von der Batterie bis zum Eingang des Wechselrichters misst (entweder direkt von Pluspol-Batterie zu Plus-Eingang am WR, oder aber als Differenz Batteriespannung (26,6V) - Spannung von Minuspol-Batterie zu Pluspol (25,6V), kannst du dies in dem Faktor als 0.0217 (1V/46A) berücksichtigen. Dann rechnet die openDTU die Verliuste auf der Leitung wieder raus und weiß, dass die Batterie nicht 25,6V sondern noch 26,6V hat.

Alternativ empirisch, Last anlegen, Batteriespannung extern mit dem Multimeter messen und den Faktor in der Software anpassen, bis beide das gleiche anzeigen.


Bzw. du hast ja Zahlen genannt.
25,6V Abschaltspannung, kurz darauf misst der Wechselrichter wieder 26,1V. Also hast du bei 900W (~35A) 0,5V Spannungsabfall auf der Leitung. Macht einen Leitungswiderstand von 0.01428 Ohm. 
Das ist meiner Meinung nach relativ viel, bedeutet es doch, dass du bei 900W Einspeisung (und 35A Stromfluss) bereits 17W Verlustleistung auf den Kabeln und Verbindungen hast.

**Reply by swingstate** [reply]
Danke, ein starker Post!
Momentan messen ich mit einem Shunt, der zwischen Batterie und der Busbar steckt. Und mit dem WR selbst. Vom Batteriepol geht jeweils ein ca. 1,5m langes 16mm2 Kabel zum Shunt bzw. zur Sicherung. Die 6mm2 Kabel zum WR sind etwas länger. Beide messen nahezu identisch, wobei der Shunt 2 Nachkommastellen zeigt. 

Wenn ich zurück zu Hause bin werden ich das mit Last genauer untersuchen und posten.
